tests/services/test_genre_detector.py

```
@@ -0,0 +1,85 @@
+import unittest
+from unittest.mock import patch, MagicMock
```

Ruff (0.8.0)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)